cmd/registry.go

```diff
@@ -32,31 +32,64 @@ type Registry struct {
 	Specs  []*bundle.Spec
 }
 
+// Default registry configuration section
+var defaultLocalOpenShiftConfig = registries.Config{
+	Namespaces: []string{"openshift"},
+	Name:       "lo-openshift",
```

Comment:
Not a fan of using `lo` here, usually refers to `loopback` (at least in the context of Linux networking).


If we use `local_openshift` as the name, it might make it a bit easier on users who added the registry by typing in `sbcli registry add local_openshift`. They would be able to type a similar command `sbcli registry remove local_openshift` to remove.